### PR TITLE
Kafka - fix escaping

### PR DESCRIPTION
**Pull request by bostko**
Addressed issues fixed

### CODE REVIEW

File: software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java [hunk 1]
```diff
@@ -125,11 +125,17 @@ public Void call() {
...
 
-        KafkaSupport support = new KafkaSupport(cluster);
+        final KafkaSupport support = new KafkaSupport(cluster);
 
         support.sendMessage("brooklyn", "TEST_MESSAGE");
-        Thread.sleep(Duration.seconds(5).toMilliseconds());
-        String message = support.getMessage("brooklyn");
-        assertEquals(message, "TEST_MESSAGE");
+
+        Asserts.succeedsEventually(MutableMap.of("timeout", Duration.FIVE_SECONDS), new Callable<Void>() {
```
Comment (neykov) [inline software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java]: Can use `Runnable` in this case, won't need `return null` further down.

**Comment by neykov** [comment]
Looks good for merge.

**Comment by neykov** [comment]
@bostko could you trigger a rebuild

File: software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java [hunk 2]
```diff
@@ -125,11 +125,16 @@ public Void call() {
...
 
-        KafkaSupport support = new KafkaSupport(cluster);
+        final KafkaSupport support = new KafkaSupport(cluster);
 
         support.sendMessage("brooklyn", "TEST_MESSAGE");
-        Thread.sleep(Duration.seconds(5).toMilliseconds());
-        String message = support.getMessage("brooklyn");
-        assertEquals(message, "TEST_MESSAGE");
+
+        Asserts.succeedsEventually(MutableMap.of("timeout", Duration.FIVE_SECONDS), new Runnable() {
```
Comment (aledsage) [inline software/messaging/src/test/java/brooklyn/entity/messaging/kafka/KafkaIntegrationTest.java]: My personal preference is to not bother configuring an explicit timeout (so it defaults to 30 seconds). Given they are not performance tests, I don't see the point in having a short(ish) timeout. On low-powered asf builds we've seen some unit tests take several seconds.

Happy to merge as-is. You've improved it over the 5 second sleep, and left the time as it was. And this is an integration test, so shouldn't fail on a low-powered asf build server.

**Comment by aledsage** [comment]
Thanks @bostko - and thanks @neykov for reviewing. Merging now.

**Pull request merged**